lib/commons/aria/index.js

```diff
@@ -1446,3 +1910,99 @@ lookupTable.role = {
 		type: 'abstract'
 	}
 };
+// extend lookupTable.role with a non-enumerable custom property to hold element/ tag names that can have any role
+Object.defineProperty(role, 'NO_ROLE', {
```

Comment:
Maybe make these separate properties on the lookup table - I don't think hiding them with `enumerable: false ` is the right way to go about it.